Tests/SonarQube.MSBuild.Tasks.IntegrationTests/Infrastructure/BuildUtilities.cs

```diff
@@ -59,7 +59,7 @@ public static ProjectRootElement CreateValidNamedProjectRoot(TestContext testCon
         /// </summary>
         public static ProjectDescriptor CreateValidProjectDescriptor(string parentDirectory)
         {
-            return CreateValidNamedProjectDescriptor(parentDirectory, "MyProject.xproj");
+            return CreateValidNamedProjectDescriptor(parentDirectory, "MyProject.xproj.txt");
```

Comment:
".txt" is more convenient when working with the unit test results in VS (clicking on the link to an ".xproj" file closes the current solution, where clicking on a ".txt" just opens the file).